### PR TITLE
responsive grid and effect

### PR DESCRIPTION
**Pull request by Alexandru-Bulai**
# PR 07-01-24

## Changes

- Make grid responsive on all screeen 
- Add transition effect and shadow to each grid-block

Jan-07-2024 13-11-32

### CODE REVIEW

File: home.html [hunk 1]
```diff
@@ -62,76 +62,147 @@
...
+                    shadow
+                    transition duration-300 
+                    ease-in-out 
+                    transform 
+                    hover:-translate-y-1 
+                    hover:shadow-xl
+                    ">
             <div class="grid__imgContainer">
-                <img src="/Assets/images/home-grid/flexibility.svg" alt="Dog face" />
+                <img class:"" src="/Assets/images/home-grid/flexibility.svg" alt="Dog face" />
```
Comment (vladiiancu) [inline home.html]: Typo? With the : character?

**Review by vladiiancu — CHANGES_REQUESTED** [review]

File: home.html [hunk 2]
```diff
@@ -143,7 +214,7 @@
     <div class="footer">
         <div class="footer__top">
             <div class="footer__logo">
-                <img src="/Assets/images/home-footer/logo_footer.svg" alt="Company logo" />
+                <img class:"" src="/Assets/images/home-footer/logo_footer.svg" alt="Company logo" />
```
Comment (vladiiancu) [inline home.html]: This is still here

**Review by vladiiancu — CHANGES_REQUESTED** [review]

File: home.html (hunk 2 above)
Comment (Alexandru-Bulai) [reply home.html]: Line 217 is part of the footer section, i actually took care of that on the other pull request :)

Comment (Alexandru-Bulai) [reply home.html]: The only reason why i didnt change that  in this PR is because i wanted to avoid conflicts, if that make sense?

**Review by vladiiancu — APPROVED** [review]

**Pull request merged**